junit-runtime/src/main/scala/org/scalajs/junit/JUnitTask.scala

```diff
@@ -0,0 +1,69 @@
+package org.scalajs.junit
+
+import com.novocode.junit.{Ansi, RichLogger}
+import Ansi._
+import sbt.testing._
+import org.scalajs.testinterface.TestUtils
+import scala.util.{Try, Success, Failure}
+
+final class JUnitTask(val taskDef: TaskDef, runner: JUnitBaseRunner)
+    extends sbt.testing.Task {
+
+  def tags: Array[String] = Array.empty
+
+  def execute(eventHandler: EventHandler, loggers: Array[Logger],
+      continuation: Array[Task] => Unit): Unit = {
+
+    val richLogger = new RichLogger(loggers, runner.runSettings,
+        taskDef.fullyQualifiedName)
+
+    if (runner.runSettings.verbose)
+      richLogger.info(c("Test run started", INFO))
+
+    val startTime = System.nanoTime
+    val tasks = execute(eventHandler, loggers)
+
+    if (runner.runSettings.verbose) {
+      val time = System.nanoTime - startTime
+      val failed = runner.taskFailedCount()
+      val ignored = runner.taskIgnoredCount()
+      val total = runner.taskTotalCount()
+      val msg = Seq(
+          c("Test run finished:", INFO),
+          c(s"$failed failed,", if (failed == 0) INFO else ERRCOUNT),
+          c(s"$ignored ignored,", if (ignored == 0) INFO else IGNCOUNT),
+          c(s"$total total,", INFO),
+          c(s"${time.toDouble / 1000000000}s", INFO))
+      richLogger.info(msg.mkString(" "))
+    }
+
+    continuation(tasks)
+  }
+
+  def execute(eventHandler: EventHandler,
+      loggers: Array[Logger]): Array[Task] = {
+
```

Comment:
The synchronous `execute` is never called for JavaScript. I can just throw `???`.